index.html

```diff
@@ -341,19 +342,25 @@ <h2><code>Window</code> interface extensions</h2>
 </pre>
     <p>Each <a>Window</a> has:</p>
     <ul>
-      <li>A <dfn>list of idle request callbacks</dfn>. The list MUST be
-      initially empty and each entry in this list is identified by a number,
-      which MUST be unique within the list for the lifetime of the <code>Window</code>
-      object.</li>
-      <li>A <dfn>list of runnable idle callbacks</dfn>. The list MUST be
-      initially empty and each entry in this list is identified by a number,
-      which MUST be unique within the list of the lifetime of the <code>Window</code>
-      object.</li>
       <li>An <dfn>idle callback identifier</dfn>, which is a number which MUST
       initially be zero.</li>
+    </ul>
+
+    <p>In addition, each <a>similar-origin window agent</a> has:</p>
+    <ul>
+      <li>A <dfn>list of idle request callbacks</dfn>. The list MUST be
+      initially empty and each entry in this list is identified by <a>Window</a>
+      and a number, the combination of which MUST be unique within the list for
+      the lifetime of the <a>similar-origin window agent</a>.</li>
```

Comment:
The second requirement should probably fall out of how you mutate the list and not be redundantly required here (a note indicating the uniqueness might be nice though). Same below.